### PR TITLE
Add bundle task to browsersync in gulpfile

### PR DESCRIPTION
**Pull request by hassony2**
- Add bundle task before as dependency from browsersync so that bundle is done automatically before browsersync first load
- Add basic error handling for browserify gulp task

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -5,6 +5,8 @@ var browserify = require('browserify'); // Providers "require" support, CommonJS
 var browserSync = require('browser-sync');
 var gutil = require('gulp-util');
 var fs = require('fs');
+var chalk = require('chalk');
+var gutil = require('gulp-util');
```
Comment (deterralba) [inline gulpfile.js]: Already there (line 6)

Comment (hassony2) [inline gulpfile.js]: Fixed.

**Comment by deterralba** [comment]
:+1: 
I don't get what chalk does ?

**Comment by hassony2** [comment]
Chalk just adds the color red to the log in the terminal

File: gulpfile.js [hunk 2]
```diff
@@ -14,7 +14,11 @@ function bundle() {
 }
 
 gulp.task('bundle', function() {
-    bundle();
+    var bundle = b.bundle();
```
Comment (deterralba) [inline gulpfile.js]: b.bundle ?

**Pull request merged**